### DISCUSSION TITLE
Problem with the tex color in ->color() and extraAttributes()

### DISCUSSION THREAD

**Discussion by micraux** [filamentphp/filament#8450] [Help] [answered]
I don't know what's happening to me but I have some trouble with the text color in a table...

1. `color('primary')` doesn't work
2. `extraAttributes(['class' => 'bg-primary-500'])` works
3. `extraAttributes(['class' => 'text-primary-500'])` doesn't work

Am I doing something wrong?

**Comment by leandrocfe** [comment]
`TextColumn` ?
`->color('primary')` should work. What about `->color('danger')` ?

**Reply by micraux** [reply]
Yes, it's on the `TextColumn`
Same with `danger`, `info`, `success`...
I really don't understand

**Reply by leandrocfe** [reply]
Fresh install? Any particular config?

**Comment by micraux** [comment]
Well, not any particular config as far as I can remember.

But I tried with a fresh install and I have some strange things:

`->color()` is working with all Filament named colors (danger, success...)

`->extraAttributes(['class' => 'bg-xxx-999])` only works when xxx = `primary` and 999 = `500` or `600` and no other defined color nor "intensity"

When I inspect the code, I can see all named color and "intensities" declared in `:root`

Is this the normal behavior?

**Reply by leandrocfe** [reply, marked as answer]
Create a [custom-theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)
You will be able to customize css classes

**Reply by micraux** [reply]
Ok, that works, thanks a lot!

So, don't you think the documentation should mention it?
https://filamentphp.com/docs/3.x/tables/columns/text#customizing-the-color

**Comment by micraux** [comment]
In my real project (not the fresh install), I already had 2 panels (`admin` and `app`) with custom theme, and thanks to your help I managed to see what was wrong.

In fact, I tried to use the `resources/css/filament/app.css` as a common css files for both themes.

To do so, I added `@import '../../app.css'` to each `theme.css` file of my 2 panels, which was working!

You pointed me to this direction and I remove my "_common css file_" idea: that works now!

By the way, is there a possibility to achieve this "_common css file_" idea: sharing one css file for all the panels and using each `theme.css` file to add more styles per panel?

Or should I open a new discussion?

**Reply by micraux** [reply]
Forget about it: I had a `.fi-ta-text-item { @apply dark:text-gray-200 !important }` in my css!

I feel dumb, sorry about that!

And by the way, the "_common css file_" idea works now perfectly of course...

I close this discussion: shame on me!
